uvloop/loop.pyx

```diff
@@ -3149,6 +3149,19 @@ cdef class Loop:
         await waiter
         return udp, protocol
 
+    def _monitor_fs(self, path: str, callback) -> asyncio.Handle:
+        cdef:
+            UVFSEvent fs_handle
+            char* c_str_path
+
+        self._check_closed()
+        fs_handle = UVFSEvent.new(self, callback, None)
+        p_bytes = path.encode('UTF-8')
+        c_str_path = p_bytes
```

Comment:
Ha, I didn't know this works...Does it correctly allocate/deallocate memory or it simply references the underlying bytes pointer?

Reply:
It's a referencing, using `PyBytes_AsStringAndSize()`.